plugins/inventory/zabbix_inventory.py

```diff
@@ -0,0 +1,380 @@
+            selectTags:
+                type: str
+                description:
+                    - query
+                    - Return a tags property with host tags.
+                    - To return all values specify 'extend'
+                    - Can be limited to different fields for example setting the vaule to ['name'] will only return the name
+                    - Additional fields can be specified by comma seperated value ['name', 'field2']
+                    - Please see U(https://www.zabbix.com/documentation/current/manual/api/reference/host/object#host_tag) for more details on field names
+            selectInheritedTags:
+                type: str
+                description:
+                    - query
+                    - Return an inheritedTags property with tags that are on all templates which are linked to host.
+                    - To return all values specify 'extend'
+                    - Can be limited to different fields for example setting the vaule to ['name'] will only return the name
+                    - Additional fields can be specified by comma seperated value ['name', 'field2']
+                    - Please see U(https://www.zabbix.com/documentation/current/manual/api/reference/host/object#host_tag) for more details on field names
+            selectTriggers:
+                type: str
+                description:
+                    - query
+                    - Return a triggers property with host triggers.
+                    - To return all values specify 'extend'
+                    - Can be limited to different fields for example setting the vaule to ['name'] will only return the name
+                    - Additional fields can be specified by comma seperated value ['name', 'field2']
+                    - Please see U(https://www.zabbix.com/documentation/current/manual/api/reference/host/object#host_tag) for more details on field names
+    login_user:
+        description:
+            - Zabbix user name.
+        type: str
+        required: true
+    login_password:
+        description:
+            - Zabbix user password.
+        type: str
+        required: true
+    http_login_user:
+        description:
+            - Basic Auth login
+        type: str
+    http_login_password:
+        description:
+            - Basic Auth password
+        type: str
+    timeout:
+        description:
+            - The timeout of API request (seconds).
+        type: int
+        default: 10
+    validate_certs:
+      description:
+       - If set to False, SSL certificates will not be validated. This should only be used on personally controlled sites using self-signed certificates.
+      type: bool
+      default: yes
+    add_zabbix_groups:
+      description:
+       - If set to True, hosts will be added to groups based on their zabbix groups
+      type: bool
+      default: false
+extends_documentation_fragment:
+    - constructed
+    - inventory_cache
```

Comment:
Wasn't working like this? You wouldn't have to specify `*_login_*, server, timeout, validate_ssl`

```suggestion
extends_documentation_fragment:
    - constructed
    - inventory_cache
    - zabbix
```

Reply:
@ttestscripting just curious if this would work. Have you tried it?

Reply:
That will work as well I am good with either method :)

Reply:
On the other hand, that doc fragment copies state of ZabbixBase and since this plugin is not using that, it would be safer to also not use doc_fragment if we decide to change something in the future that will not be reflected in this plugin.

Maybe eventually we will rework our base class as there already was an initiative for that in the past and then we will be able to migrate this plugin to use that also without any hussle